guides/plugins/plugins/administration/add-rule-assignment-configuration.md

```diff
@@ -0,0 +1,270 @@
+# Add rule assignment configuration
```

Comment:
New files have to be added to the SUMMARY.md file in order to show up in search results and the navigation